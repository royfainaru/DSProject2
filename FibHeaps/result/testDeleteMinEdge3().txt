╮
╰── ╭────────────────╮
    │ Key: 2         │
    ├────────────────┤
    │ Rank: 0        │
    │ Marked: false  │
    │ Parent: (null) │
    │ Next: (null)   │
    │ Prev: (null)   │
    │ Child: (null)  │
    ╰────────────────╯
