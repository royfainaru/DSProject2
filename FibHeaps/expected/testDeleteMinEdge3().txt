╮
╰── ╭────────────────╮
    │ Key: 2         │
    ├────────────────┤
    │ Rank: 0        │
    │ Marked: false  │
    │ Parent: (null) │
    │ Next: 2        │
    │ Prev: 2        │
    │ Child: (null)  │
    ╰────────────────╯
